### PR TITLE
Readme update - Closes #341

### LINKED ISSUE #341: Documentation: legacy RPC procedure names
Update documentation so the procedure names are up-to-date:

- `get.voters` -> `get.votes_received`
- `get.votes` -> `get.votes_sent`

Also:

- Missing MySQL installation instruction

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #341

### How was it solved?

- [x] Update RPC names
- [x] Add MySQL installation docs

### CODE REVIEW

File: docs/api/websocket_json_rpc_api.md [hunk 1]
```diff
@@ -224,7 +227,7 @@ Get address with certain Lisk account ID
 
 > TODO: Add some more examples
 
-### get.votes
+### get.votes_sent
 
 Retrieves votes for a single account based on address, public key or delegate name.
```
Comment (chris529) [inline docs/api/websocket_json_rpc_api.md:232]:
```suggestion
Retrieves votes for a single account based on address, public key, or delegate name.
```

File: docs/config_options.md [hunk 2]
```diff
@@ -68,6 +68,12 @@ LISK_CORE_CLIENT_TIMEOUT=30            # Lisk Core client timeout (in seconds)
 # Local Redis cache for Lisk microservice
 # Note it is a different DB that SERVICE_BROKER uses
```
Comment (chris529) [inline docs/config_options.md:69]:
```suggestion
# Note: SERVICE_BROKER uses a different DB
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a couple of minor points 👍

File: docs/api/websocket_json_rpc_api.md [hunk 3]
```diff
@@ -6,32 +6,35 @@ The Lisk Service is a web application that interacts with the whole Lisk ecosyst
...
+- [2.5. Transactions](#25-transactions)
+	- [2.5.1. get.transactions](#251-gettransactions)
+	- [2.5.2. get.transactions.statistics.day](#252-gettransactionsstatisticsday)
+	- [2.5.3. get.transactions.statistics.month](#253-gettransactionsstatisticsmonth)
+- [2.6. Network](#26-network)
+	- [2.6.1. get.network.status](#261-getnetworkstatus)
+	- [2.6.2. get.network.statistics](#262-getnetworkstatistics)
+	- [2.6.3. get.search](#263-getsearch)
+
+<!-- /TOC -->
```
Comment (Tschakki) [inline docs/api/websocket_json_rpc_api.md:37]: Just a remark: If you would write your Readme files in assciidoc, you can autogenerate those table of contents by adding a simple `:toc:` attriubte to the page :)

**Review by Tschakki — APPROVED** [review]

File: docs/api/websocket_json_rpc_api.md (hunk 3 above)
Comment (MichalTuleja) [reply docs/api/websocket_json_rpc_api.md:37]: Thanks Mona, does it help you when I add the following code block?
```
<!-- AsciiDoc TOC marker
:toc:
-->
```

Comment (Tschakki) [reply docs/api/websocket_json_rpc_api.md:37]: No, there is no help for me, it was just a recommendation that would make your life easier writing READMEs ;)

Comment (MichalTuleja) [reply docs/api/websocket_json_rpc_api.md:37]: I need them for Markdown anyway. But thanks for the recommendation.

**Pull request merged**